.gitlab-ci.yml

```diff
@@ -13,7 +13,7 @@ default:
       - api_failure
 
 .docker-env:                       &docker-env
-  image:                           "paritytech/ci-linux:production"
+  image:                           "paritytech/ci-linux:staging"
```

Comment:
Is this a temporary fix or longterm?

Reply:
Actually we can leave `staging` image. It usually contains latest stable rust. 'production' one upgraded ones it fully complaint with substrate, polkadot and cumulus, which has some lag.